backend/api/Controllers/Models/CustomMissionQuery.cs

```diff
@@ -8,7 +8,8 @@ public struct CustomInspectionQuery
 
         public float? VideoDuration { get; set; }
 
-        public string? AnalysisTypes { get; set; }
+        public AnalysisType? AnalysisType { get; set; }
+
```

Comment:
```suggestion
```
Remove this newline for consistent formatting

Reply:
Done !